strawberry/http/async_base_view.py

```diff
@@ -22,6 +22,7 @@
 from strawberry.types import ExecutionResult
 from strawberry.types.graphql import OperationType
 
+from ..types.context_wrapper import ContextWrapper
```

Comment:
I think it is ok (and usually nice) to use relative imports for something already inside it (i.e. using `from .<something>`), but when importing from a parent (i.e. `from ..<something>) I think it starts to get messy.

In this case I would suggest importing from `strawberry.types.context_wrapper`

Reply:
done